frontend/src/routes/surveyfeedback/+page.svelte

```diff
@@ -21,17 +24,11 @@
 		'This is an overview over which milestones for the current survey have been completed';
 </script>
 
-<AbstractContent
-	showBottomNavbar={true}
-	lastpage="{base}/dataAcquisition/surveyC"
-	nextpage="{base}"
-	infopage="{base}/info"
->
-	<TableDisplay
-		data={data_to_display}
-		{statusIndicator}
-		{caption}
-		statusColumns="{['status']},"
-		searchableColumns={['name']}
-	/>
-</AbstractContent>
+<TableDisplay
+	data={data_to_display}
+	{statusIndicator}
+	{caption}
+	{celllinks}
+	statusColumns="{['status']},"
```

Comment:
why is this quoted vs searchableColumns below, are they different?

Reply:
mistake that happened due to autoformatting when a comma is accidentally added at the end.